src/components/Table/Table.stories.jsx

```diff
@@ -387,8 +416,97 @@ export const Pagination = () => {
 
 Pagination.storyName = 'Pagination with React-Table';
 
-// Export const StructuredView = () => {};
-// StructuredView.storyName = 'Structured View with React-Table';
-//
-// export const KitchenSink = () => {};
+export const StructuredView = () => {
+    const data = React.useMemo(() => tableDataNested, []);
+    const columns = React.useMemo(() => [
+        {
+            Header: 'Name',
+            id: 'name',
+            accessor: row => row.name.value,
+            Cell: cellInfo => {
+                const {row} = cellInfo;
+                return (
+                    <ExpansionContainer
+                        canExpand={row.canExpand}
+                        isExpanded={row.isExpanded}
+                        depth={row.depth}
+                        getToggleRowExpandedProps={row.getToggleRowExpandedProps}
+                    >
+                        <ListItem
+                            iconSize="default"
+                            typographyVariant="body"
+                            iconStart={row.original.name.icon}
+                            label={row.values.name}
+                        />
+                    </ExpansionContainer>
```

Comment:
Can we embed this to the `TableCell` component and passing the needed props to it, to avoid exposing `ExpansionContainer` and `ListItem` and simplify the implementation of the structured view.

Reply:
Unfortunately, because `TableCell` wraps the rendered value in the cell, it's difficult to render the chevron right/down icons in the correct place in relation to the content type icon. So, in this case, this `ExpansionContainer` that's rendered, gets wrapped by the `TableCell`. Maybe we can have a quick call about this so that I can explain it better.